<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>luamwindow</class>
 <widget class="QMainWindow" name="luamwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1025</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Audioshield Folder:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbSelectASFolder">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Folder</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leAudioshieldFolder"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Proto File:              </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbOpenProtoFileSelector">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Folder</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leProtoFile"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Output Folder:       </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbSelectOutputFolder">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Folder</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leOutputFolderFolder"/>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>File Name:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leOutputFilename"/>
        </item>
        <item>
         <widget class="QPushButton" name="pbMergeToOutput">
          <property name="text">
           <string>Merge</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QWidget" name="">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QTableView" name="tbvFolders">
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QPushButton" name="pbAddFolder">
              <property name="text">
               <string>Add Folder</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pbRemoveFile">
              <property name="text">
               <string>Remove Folder</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QTableView" name="tbvFiles">
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QPushButton" name="pbUp">
              <property name="text">
               <string>Up</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pdDown">
              <property name="text">
               <string>Down</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pbAddFiles">
              <property name="text">
               <string>AddFiles</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pbDisable">
              <property name="text">
               <string>Disable</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pbDelete">
              <property name="text">
               <string>Delete</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1" rowspan="2">
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Push it to:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbPushIntoEasy">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Easy</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbPushIntoHard">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Hard</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbPushIntoElite">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Elite</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbPushIntoCasual">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Casual</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbPushIntoLegacyHard">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Legacy hard</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>503</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QTextEdit" name="edtDiag">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>pbPushIntoEasy</zorder>
   <zorder>pbPushIntoHard</zorder>
   <zorder>pbPushIntoElite</zorder>
   <zorder>pbPushIntoCasual</zorder>
   <zorder>pbPushIntoLegacyHard</zorder>
   <zorder>label</zorder>
   <zorder>lvSets</zorder>
   <zorder>pbUp</zorder>
   <zorder>pdDown</zorder>
   <zorder>pbDisable</zorder>
   <zorder>pbDelete</zorder>
   <zorder>verticalSpacer</zorder>
   <zorder>leAudioshieldFolder</zorder>
   <zorder>pbSelectASFolder</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>horizontalSpacer</zorder>
   <zorder>verticalSpacer_2</zorder>
   <zorder>tbvFolders</zorder>
   <zorder>tbvFiles</zorder>
   <zorder>pbAddFolder</zorder>
   <zorder>pbRemoveFile</zorder>
   <zorder>verticalSpacer_3</zorder>
   <zorder>label</zorder>
   <zorder>pbPushIntoEasy</zorder>
   <zorder>edtDiag</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1025</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
